### ISSUE TITLE
UsableNet - WCAG Team comment - Consider the legal and practical importance of conformance.

### ISSUE THREAD

**Issue by UsableNet-WCAG** [w3c/wcag3#258]
Reviewing 3.0 in its current form indicates some significant changes to terminology , particularly in the area of conformance. With the growing US legal activity focused on compliance there is a need to quickly and cheaply provide evidence of conformance for any size business.  Additional detail on how conformance could or should be presented would aid companies looking to communicate their efforts and status.

**Comment by hidde** [comment]
Thanks for this comment! Closing this issue as it relates to a much earlier version of Silver/WCAG 3. We've just published the latest [working draft](https://www.w3.org/TR/wcag-3.0/), which features a new model and has had legal aspects considered within the conformance subgroup that worked on it. We welcome  feedback on these new sections (Conformance and Reporting).
